docs/source/security/authentication.mdx

```diff
@@ -97,13 +110,13 @@ For our first example, let's look at a resolver that's only accessible with a va
 
 ```js
```

Comment:
Do we want `ts` everywhere?

This is another "floating" property that's invalid syntax by itself. If we think that's fine we can leave it (and also possibly ignore my comment earlier).

Reply:
This should be TS thank you! For this snippet specifically, I think the invalid syntax is okay because the code is describing logic, and we wouldn't actually expect anyone to copy and paste this somewhere. (I'll add `disableCopy` to the code block, too—because this is a valid concern) 

Your former comment about the last code snippet still stands, though! I think it's reasonable to have snippets we expect folks to directly copy be valid syntax. 👏